pkg/api/mlflow/service/artifact/validators_test.go

```diff
@@ -139,3 +139,156 @@ func TestValidateListArtifactsRequest_Error(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateGetArtifactRequest_Ok(t *testing.T) {
+	testData := []struct {
```

Comment:
sometimes we use `testCases` sometimes `testData`, can you please unity it? for me `tests` just works really well.

Reply:
done